types/console-log-tree/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for console-log-tree 1.2
+// Project: https://github.com/QiShaoXuan/log-tree#readme
+// Definitions by: Rajas Paranjpe <https://github.com/ChocolateLoverRaj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace library;
+
+export interface Tree {
+  name: string;
+  children?: readonly Tree[];
+}
+
+export  function parse(tree: Tree | readonly Tree[]): string;
```

Comment:
```suggestion
export function parse(tree: Tree | readonly Tree[]): string;
```

Reply:
AFAIK those should have optional params (missing from readme documentation):
https://github.com/QiShaoXuan/log-tree/blob/f4e39c4222a4e6caf60ab7f9493eaa1258249ac7/src/index.js#L20